src/NeoSharp.Application/Client/Prompt[Wallet].cs

```diff
@@ -71,6 +71,21 @@ private void ImportNep2(string nep2key)
                 _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
             }
         }
+        
+        [PromptCommand("create account", Category = "Account", Help = "Create a new account")]
+        private void AccountCreateCommand()
+        {
+            try
+            {
+                SecureString secureString = _consoleReader.ReadPassword();
+                _walletManager.CreateAccount(secureString);
+            }
+            catch(System.Exception exception)
```

Comment:
Remove Exception, and check if is open the wallet for show a custom exception, the others, could be captured for Prompt

Reply:
I didn't understand the "check if is open the wallet", CreateAccount is already checking it